fixit/rules/use_assert_in.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.helpers import ensure_type
+
+from fixit.common.base import CstLintRule
+from fixit.common.utils import InvalidTestCase as Invalid, ValidTestCase as Valid
```

Comment:
Simply import from fixit to be simpler. It's a convention we used.
```suggestion
from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid
```
@dkteo can you also fix this, then I can merge for you.

Reply:
done!